packages/mongodb_atlas/data_stream/mongod_audit/fields/fields.yml

```
@@ -0,0 +1,63 @@
+- name: mongodb_atlas.mongod_audit
+  type: group
+  fields:
+    - name: hostname
+      type: keyword
+      description: Human-readable label that identifies the host that stores the log files that you want to download.
+    - name: local
+      type: group
+      fields:
+        - name: ip
+          type: ip
+          description: A document that contains the IP address of the running instance.
+        - name: is_system_user
+          type: boolean
+          description: This field indicates whether the user who caused the event was a system user.
```

```suggestion
          description: True if the event is caused by a system user, false otherwise.
```
Applies to the remote user description as well.